### PR TITLE
Dpl-733 ontflowcell update

### PR DESCRIPTION
**Pull request by seenanair**
Removed bootstrap b-input and b-invalid-feedback with traction-input and traction-field-error components

### CODE REVIEW

File: src/components/ont/runs/ONTFlowcell.vue [hunk 1]
```diff
@@ -66,31 +76,48 @@ export default {
...
+          : 'Enter at valid Flowcell ID (3 letters then at least 3 numbers)'
       }
+      return ''
+    },
+    /**Returns a message if invalid otherwise empty */
+    barcodeValidationError() {
+      return this.barcode ? (this.barcodeState ? '' : 'Enter a valid Pool Library barcode') : ''
     },
+    /** Is the flowcellId empty or not */
+    isFlowIdExists() {
```
Comment (stevieing) [inline src/components/ont/runs/ONTFlowcell.vue:99]: Should this be isFlowcellIdExists?

File: tests/unit/components/ont/runs/ONTFlowcell.spec.js [hunk 2]
```diff
@@ -54,6 +56,7 @@ describe('ONTFlowcell', () => {
       expect(ontFlowcell.poolTubeBarcode).toBeDefined()
       expect(ontFlowcell.poolTubeBarcode).toEqual('TRAC-1-A')
       expect(ontFlowcell.flowcell_bg_colour).toBeDefined()
+      console.log(ontFlowcell.flowcell_bg_colour)
```
Comment (stevieing) [inline tests/unit/components/ont/runs/ONTFlowcell.spec.js:59]: Can this be removed?

**Review by stevieing — CHANGES_REQUESTED** [review]
Great to see the last bootstrap component removed from index.js

File: src/components/ont/runs/ONTFlowcell.vue (hunk 1 above)
Comment (seenanair) [reply src/components/ont/runs/ONTFlowcell.vue]: Yes, sure , that make it more clear . Changed now

File: tests/unit/components/ont/runs/ONTFlowcell.spec.js (hunk 2 above)
Comment (seenanair) [reply tests/unit/components/ont/runs/ONTFlowcell.spec.js]: 👍

**Review by stevieing — APPROVED** [review]

**Pull request merged**